ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -1062,6 +1139,9 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
         // Minimum padding from top when the sheet is fully expanded
         static let minimumTopExpandedPadding: CGFloat = 25.0
 
+        // The padding allocated to the space between the sheet and the edge when attached to the leading or trailing edge
+        static let horizontalSheetPadding: CGFloat = 8
```

Comment:
Since fluent 2 we try to use our tokens whenever possible so this should be:
`static let horizontalSheetPadding: CGFloat = GlobalTokens.spacing(.size80)`